modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/asset_list/scope.jsp

```diff
@@ -72,48 +73,127 @@ List<Group> selectedGroups = editAssetListDisplayContext.getSelectedGroups();
+			message='<%= LanguageUtil.get(request, "other-site-or-asset-library") + StringPool.TRIPLE_PERIOD %>'
+			method="get"
+			url="javascript:void(0);"
+		/>
+	</liferay-ui:icon-menu>
 </c:if>
 
 <aui:script use="liferay-search-container">
-	const searchContainer = Liferay.SearchContainer.get(
+	var searchContainer = Liferay.SearchContainer.get(
 		'<portlet:namespace />groupsSearchContainer'
 	);
 
 	searchContainer.get('contentBox').delegate(
 		'click',
 		(event) => {
-			const link = event.currentTarget;
+			var link = event.currentTarget;
 
-			const tr = link.ancestor('tr');
+			var tr = link.ancestor('tr');
 
 			searchContainer.deleteRow(tr, link.getAttribute('data-rowId'));
 
 			searchContainer.updateDataStore();
 
-			const groupIds = document.getElementById(
-				'<portlet:namespace />groupIds'
-			);
-
-			if (groupIds) {
-				const searchContainerData = searchContainer.getData();
-
-				groupIds.setAttribute('value', searchContainerData.split(','));
-
-				submitForm(document.<portlet:namespace />fm);
-			}
+			updateGroupIds();
 		},
 		'.modify-link'
 	);
+
+	var selectManageableGroupIcon = document.getElementById(
+		'<portlet:namespace />selectManageableGroup'
+	);
+
+	if (selectManageableGroupIcon) {
+		selectManageableGroupIcon.addEventListener('click', (event) => {
+			event.preventDefault();
+
+			Liferay.Util.openSelectionModal({
+				id: '<%= editAssetListDisplayContext.getSelectGroupEventName() %>',
+				onSelect: function (selectedItem) {
+					var entityId = selectedItem.groupid;
+
+					var searchContainerData = searchContainer.getData();
+
+					if (searchContainerData.indexOf(entityId) == -1) {
+						<portlet:namespace />addRow(
+							entityId,
+							selectedItem.groupdescriptivename,
+							selectedItem.groupscopelabel
+						);
+					}
+				},
+				selectEventName:
+					'<%= editAssetListDisplayContext.getSelectGroupEventName() %>',
+				title: '<liferay-ui:message escape="<%= true %>" key="scopes" />',
+				url: '<%= editAssetListDisplayContext.getGroupItemSelectorURL() %>',
+			});
+		});
+	}
+
+	window['<portlet:namespace />addRow'] = function (groupId, name, scopeLabel) {
+		var data = searchContainer.getData(true);
+		if (data.includes(groupId)) {
+			return;
+		}
+
+		var rowColumns = [];
+
+		rowColumns.push('<span class="text-truncate">' + name + '</span>');
+		rowColumns.push(scopeLabel);
+		rowColumns.push(
+			'<a class="modify-link" data-rowId="' +
+				groupId +
+				'" href="javascript:void(0);"><%= UnicodeFormatter.toString(removeLinkIcon) %></a>'
+		);
+
+		searchContainer.addRow(rowColumns, groupId);
+
+		searchContainer.updateDataStore();
+
+		updateGroupIds();
+	};
+
+	function updateGroupIds() {
+		var groupIds = document.getElementById('<portlet:namespace />groupIds');
+
+		if (groupIds) {
+			var searchContainerData = searchContainer.getData();
+
+			groupIds.setAttribute('value', searchContainerData.split(','));
+
+			<portlet:namespace />saveSelectBoxes();
+		}
+	}
```

Comment:
Are all these changes in `asset-list-web` related to the [ticket](https://issues.liferay.com/browse/LPS-167991)?